### DISCUSSION TITLE
How to remove editor content and not let it stay after submitting a form?

### DISCUSSION THREAD

**Discussion by darrylferdinands** [ueberdosis/tiptap#818] [General Questions]
I use tiptap with a form like this :
` <TipTapForm v-model="oppInEditing.qualifications"/>`
 However, when I submit the form, the content does not get destroyed in the editor and stays there. What should I do here to make the content not stay there after I have finished submitting the form?

**Comment by hanspagel** [comment]
Try to reset the content. Using v-model it probably works like that:
```js
this.oppInEditing.qualifications = null;
```

I’ve never used it like that, so I would use the Editor instance to set the content there:

```js
this.editor.setContent(null);
```
Here is a CodeSandbox demonstrating the behaviour: https://codesandbox.io/s/vue-issue-template-forked-12p7z?file=/src/App.vue

Hope that helps!
